Directory.Packages.props

```diff
@@ -0,0 +1,59 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- prod packages -->
```

Comment:
nit: consider using a separate ItemGroup for prod and non-prod.
Then should alphabetize the packages within those groups.

Reply:
[fixed](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4298/commits/ecbbae97770b074f61c3b653223c41cb4d82b057)